persistence/mongodb-record/src/main/scala/net/liftweb/mongodb/record/field/DBRefField.scala

```diff
@@ -81,13 +81,11 @@ class DBRefField[OwnerType <: BsonRecord[OwnerType], RefType <: MongoRecord[RefT
   // assume string is json
   def setFromString(in: String): Box[DBRef] = {
     val dbo = JSON.parse(in).asInstanceOf[BasicDBObject]
-    MongoDB.use(ref.meta.connectionIdentifier) ( db => {
-      val id = dbo.get("$id").toString
-      ObjectId.isValid(id) match {
-        case true => Full(set(new DBRef(db, dbo.get("$ref").toString, new ObjectId(id))))
-        case false => Full(set(new DBRef(db, dbo.get("$ref").toString, id)))
-      }
-    })
+    val id = dbo.get("$id").toString
+    ObjectId.isValid(id) match {
+      case true => Full(set(new DBRef(dbo.get("$ref").toString, new ObjectId(id))))
+      case false => Full(set(new DBRef(dbo.get("$ref").toString, id)))
+    }
```

Comment:
We used to have to fetch the DB, but DBRef no longer supports that. So, yeah, it will always be`Full` now. Not really sure how else to handle this.